core/cmd/local_client_test.go

```diff
@@ -344,7 +344,7 @@ func TestClient_RebroadcastTransactions_BPTXM(t *testing.T) {
 	// the transaction cannot be seen from another connection.
 	config, _, cleanup := cltest.BootstrapThrowawayORM(t, "rebroadcasttransactions", true, true)
 	defer cleanup()
-	config.Config.Dialect = orm.DialectPostgres
```

Comment:
Might be worth a quick comment why `DialectPostgresWithoutLock` as I don't suspect this is obvious.

Reply:
I feel like using no lock is the obvious choice. You would need a comment to explain why a lock is necessary